drivers/boston_project_scorecard/app/views/boston_project_scorecard/warehouse_reports/scorecards/_financial_performance.haml

```diff
@@ -1,5 +1,5 @@
 %h2{id: 'financial-performance'}  Project Financial Performance
-%table.table.table-bordered
+%table.table.table-bordered{style: 'max-width: 99%'}
```

Comment:
Thisis probably fine, but I was thinking it could go in the print style sheet.  Something like `.report-name table { max-width: 99%; }`.  That might not be desirable though.